tdrs-backend/tdpservice/data_files/models.py

```diff
@@ -12,7 +13,7 @@
 def get_s3_upload_path(instance, filename):
     """Produce a unique upload path for S3 files for a given STT and Quarter."""
     return os.path.join(
-        f'data_files/{instance.stt.id}/{instance.quarter}',
+        f'{settings.APP_NAME}/data_files/{instance.stt.id}/{instance.quarter}',
```

Comment:
This works, but given the solution I found above for location in our clients, it might be cleaner to do do that if it works.